nix/pkgs.nix

```diff
@@ -35,7 +35,8 @@ let
         packages.typed-protocols.configureFlags = [ "--ghc-option=-Werror" ];
         packages.io-sim.configureFlags = [ "--ghc-option=-Werror" ];
         packages.io-sim-classes.configureFlags = [ "--ghc-option=-Werror" ];
-        packages.ouroboros-network.configureFlags = [ "--ghc-option=-Werror" ];
+        packages.ouroboros-network.configureFlags = [ "--ghc-option=-Werror" "--flags=cddl"];
+        packages.ouroboros-network.components.tests.cddl.build-tools = [pkgs.cddl pkgs.cbor-diag];
```

Comment:
@rvl I have a cabal test suite called cddl that needs external Ruby tools from the `cddl` package and the `cbor-diag` package.
Is this the right way to add the packages to the CI ?

Reply:
Hi @MarcFontaine, yes this is exactly how you would add executable dependencies in `haskell.nix`.